services/redis/queue.js

```diff
@@ -18,6 +18,18 @@ registerV2Queue.on('error', (err) => {
   console.log('error on assigning register v2 queue', err);
 });
 
+const followTopicQueue = new Bull('followTopic', connectRedis, {
+  redis: {
+    ...bullConfig,
+    maxRetriesPerRequest: 100,
+    connectTimeout: 30000
+  }
+});
+
+followTopicQueue.on('error', (err) => {
+  console.log('error on assigning register v2 queue', err);
+});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error message in the `followTopicQueue` error handler is misleading as it mentions "register v2 queue". It should be updated to reflect the correct queue name.
```diff
-  console.log('error on assigning register v2 queue', err);
+  console.log('error on assigning followTopic queue', err);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->